### ISSUE TITLE
Unable to join new master use arp mode 

### ISSUE THREAD

**Issue by offSwitch** [kube-vip/kube-vip#319]
**Describe the bug**
Unable to join new master
**To Reproduce**
I have three node dockerone(192.168.137.15) dockertwo(192.168.137.16) dockerthree(192.168.137.17)
init steps:
```shell
export VIP=192.168.137.2
export INTERFACE=ens33
KVVERSION=$(curl -sL https://api.github.com/repos/kube-vip/kube-vip/releases | jq -r ".[0].name")
alias kube-vip="ctr run --rm --net-host ghcr.io/kube-vip/kube-vip:$KVVERSION vip /kube-vip"
kube-vip manifest pod \
    --interface $INTERFACE \
    --vip $VIP \
    --controlplane \
    --services \
    --arp \
    --leaderElection | tee  /etc/kubernetes/manifests/kube-vip.yaml
```
kube-vip.yaml
```yaml
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: null
  name: kube-vip
  namespace: kube-system
spec:
  containers:
  - args:
    - manager
    env:
    - name: vip_arp
      value: "true"
    - name: port
      value: "6443"
    - name: vip_interface
      value: ens33
    - name: vip_cidr
      value: "32"
    - name: cp_enable
      value: "true"
    - name: cp_namespace
      value: kube-system
    - name: vip_ddns
      value: "false"
    - name: svc_enable
      value: "true"
    - name: vip_leaderelection
      value: "true"
    - name: vip_leaseduration
      value: "5"
    - name: vip_renewdeadline
      value: "3"
    - name: vip_retryperiod
      value: "1"
    - name: vip_address
      value: 192.168.137.2
    image: ghcr.io/kube-vip/kube-vip:v0.4.0
    imagePullPolicy: Always
    name: kube-vip
    resources: {}
    securityContext:
      capabilities:
        add:
        - NET_ADMIN
        - NET_RAW
        - SYS_TIME
    volumeMounts:
    - mountPath: /etc/kubernetes/admin.conf
      name: kubeconfig
  hostAliases:
  - hostnames:
    - kubernetes
    ip: 127.0.0.1
  hostNetwork: true
  volumes:
  - hostPath:
      path: /etc/kubernetes/admin.conf
    name: kubeconfig
status: {}
```
```shell
#execute on dockerone
kubeadm init   --pod-network-cidr=10.244.0.0/16 --control-plane-endpoint="192.168.137.2:6443"  --upload-certs
#execute on dockertwo
 kubeadm join 192.168.137.2:6443 --token sxik3s.93kozij7j3rmom9g \
        --discovery-token-ca-cert-hash sha256:f66aaaae18b575e46fd002304f19042df8dd60d400f26ce61c9c15407073862c \
        --control-plane --certificate-key b279a9aeba2e9661062aeae8074a07c5c96af4d9715813c1be499eaf15cc5fe4 \
        --v=10
```

**Expected behavior**
init correctly
**Screenshots**
![image](https://user-images.githubusercontent.com/11695496/143169715-8c9f5cc9-8a6f-4e94-99d5-e7566a7c6003.png)
![image](https://user-images.githubusercontent.com/11695496/143169749-f90a3a9e-d431-4b6f-92c4-d582a39308c6.png)
![image](https://user-images.githubusercontent.com/11695496/143169874-7e3d3fa2-f72c-43bf-aff2-956976fe55e8.png)

**Environment (please complete the following information):**
 - OS/Distro: Ubuntu 20.04
 - Kubernetes Version: v1.22.4
 - Kube-vip Version: v0.4.0

**Additional context**

**Comment by thebsdbox** [comment]
If `kubeadm init` completed succesfully then the VIP is up and connectivity has been confirmed by the `kubeadm` process.. Can you try to curl and ping the VIP from the other nodes?

**Comment by offSwitch** [comment]
@thebsdbox 
Thanks for your reply, I have solved this issue by resetting containerd. but I found a new problem.
1. After successfully initializing the three nodes cluster, after reset by kubeadm, there will be a node's etcd error which can't be reset normally, after I checked, I found that two nodes' pause containers are not cleaned normally, vip is not released normally, flannel is also not cleaned normally.
2. static pod bpg method seems to not work properly on calico network, calico-cni always fails to initialize properly, so I turned to flannel and found that flannel can be initialized properly.
My three hosts are dockerone(192.168.137.15),dockertwo(192.168.137.16),dockerthree(192.168.137.17).
pod-cid is 10.244.0.0/16 for both flannel and calico.
vip are configured 192.168.137.2
